src/core/trulens/core/schema/app.py

```diff
@@ -161,6 +149,10 @@ def __init__(
         kwargs["app_id"] = self._compute_app_id(
             kwargs["app_name"], kwargs["app_version"]
         )
+        if app_id is not None and kwargs["app_id"] != app_id:
+            raise ValueError(
```

Comment:
Can you integrated the above deleted deprecation message into this error message?